test/test_runner.rb

```diff
@@ -1,35 +0,0 @@
-require 'ci/reporter/minitest'
-
-module ForemanMiniTestRunner
```

Comment:
Nothing yet, they deprecated the whole Runner API. It's a major roadblock for plugins.